server/eclipse-project/src/main/java/us/freeandfair/corla/json/CountyDashboardRefreshResponse.java

```diff
@@ -104,19 +114,17 @@
   /**
    * The number of ballots audited.
    */
-  private final Integer my_number_of_ballots_audited;
+  private final Integer my_ballots_audited;
```

Comment:
Let's rename attrs of the form `my_number_of_XS` (`XS` is plural) to `X_count` (`X` is singular).

This would deviate from the above slightly and become `audited_ballot_count`. This will make it easier to distinguish an actual list of audited ballots from the length of such a list, just by glancing at the key name.